.gitattributes

```diff
@@ -4,4 +4,4 @@ output/gene-filtering.Rmd/eset-filtered.rdata filter=lfs diff=lfs merge=lfs -tex
 
 data/eset-filtered.rds filter=lfs diff=lfs merge=lfs -text
 output/images-cellcycleR-convergence.Rmd/*rds filter=lfs diff=lfs merge=lfs -text
-
+data/cellcycle-genes-previous-studies/* filter=lfs diff=lfs merge=lfs -text
```

Comment:
Did you want to commit these files as LFS files? Because [data/ is ignored by default](https://github.com/jdblischak/fucci-seq/blob/master/.gitignore#L1) to avoid accidentally committing large files, you would need to force add these files with `git add -f`.

If you tried to add them with `wflow_publish()`, unfortunately that currently isn't supported (the git2r package can't force add files in ignored subdirectories for some reason).

Reply:
Thanks! I forgot about that one. I just force added these by `git add -f`.